### PR TITLE
Product planning

### PR DESCRIPTION
**Pull request by sammachin**
details of what we've discussed over our previous product calls.

Still TODO is to add the Business Rules around billing but that will follow in a separate PR

### CODE REVIEW

File: product/plan.md [hunk 1]
```diff
@@ -48,5 +48,4 @@ We will build the following to **Meet the compliance needs of the enterprise**;
 
 We offer a number of ways that users can run the FlowForge product, we will always offer an open source version that has the core features. Features that offer higher business value or permit users to share across larger groups will be offered as part of our paid enterprise proposition.
 FlowForge Cloud is our hosted public cloud offering, this will generally include all features that are availble but may not offer certain features such as SSO where integration is required between the platform and an enterprise. We will offer customers the ability to have a dedicated cloud instance managed by us if thats is a deployment model they require.
-As new features are designed we will carefully consider each one as to if it fits within the opensource or commercial offering.
-
+A more detailed breakdown of the pricing and split between our Open Source and Enterprice Editions is on the [pricing pricicpes](pricing.md) page
```
Comment (knolleary) [inline product/plan.md:51]:
```suggestion
A more detailed breakdown of the pricing and split between our Open Source and Enterprise Editions is on the [pricing principles](pricing.md) page
```

**Review by knolleary — COMMENTED** [review]
Typo fix suggested, but that aside, I think this properly reflects the discussions we've had and gets it documented properly in the right place. 👍🏻

File: product/versioning.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Versioning
+
+## 1.0
+
+The 1.0 Release of FlowForge represents a significant milestone, we are making a statement to our customers about the product.
+
+Therefore the following are considerations & requirements in order to be able to stand behind that statement.
+
+- We have the admin features required to manage the platform at an appropriate scale.
```
Comment (ZJvandeWeg) [inline product/versioning.md:9]: I agree with this list. My doubt and fear is still that this becomes a large chunck of work that's not tightly enough scoped. With that we don't ship fast enough. Hope that makes sense?

I tried to circumvent this with #85. So if we timebox the release with this list, I'm supportive.

File: product/versioning.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+- We have the monitoring and alerting features to run a commercial SaaS platform.
+- We have a suitable licensing model in place
```
Comment (ZJvandeWeg) [inline product/versioning.md:11]: Minor note: for me just project pricing might even suffice for 1.0. Devices would be great, teams IMO is a stretch goal.

Reason being that building a feature gap between tiers takes time. Building a gap on 3 axis is taking exponentially more time is my gut feeling.

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: product/versioning.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+- The pricing model is sufficiently mature and implemented in the platform as to support our plans for the 1.x timeline. 
+- The devices feature is capable of supporting at least one real world use case end to end.
+- Both the Forge app and Device Agent are at 1.0 and the interface between them is locked, such that agents running 1.0 can connect to any 1.x forge application.
+
+
+### Backports
+
+Once 1.0 has been released the 0.x line will become deprecated, in general no fixes will be backported to the 0.x line, critical security updates may be considered on a case by case basis.
+
+The 1.x line will be supported for at least 12 months from the first release of 1.0, this means that customers may need to update to a 1.x release in order to receive fixes and new features. Even after the release of 2.0 fixes will be backported into the last 1.x release for the duration of the support period.
```
[the remark this answers is not in the document]
Comment (sammachin) [reply product/versioning.md:21]: yes thats better wording

**Pull request merged**